src/Model/User.php

```diff
@@ -2,11 +2,16 @@
 
 namespace UserBase\Client\Model;
 
+use LinkORB\Contracts\UserbaseRole\RoleInterface;
 use Symfony\Component\Security\Core\User\AdvancedUserInterface;
-use Symfony\Component\Security\Core\Role\Role;
 use RuntimeException;
 
-final class User implements UserInterface, AdvancedUserInterface, AccountContainerInterface, PolicyContainerInterface
+final class User implements
+    AccountContainerInterface,
+    AdvancedUserInterface,
+    PolicyContainerInterface,
+    RoleInterface,
+    UserInterface
 {
```

Comment:
Interface per line, sorted alphabetically. Add RoleInterface which this class already implements.